File: Codigo_Fonte_c/Lista 1/lista1_12.c
#include <stdio.h>
#include <math.h>

int main(){
	float a, b, c, delta, raiz_delta, raiz1, raiz2;
	scanf("%f", &a);
	scanf("%f", &b);
	scanf("%f", &c);

	delta = pow(b, 2) - (4 * a * c);
	raiz_delta = sqrt(delta);
	raiz1 = (-b + raiz_delta) / (2 * a);
	raiz2 = (-b - raiz_delta) / (2 * a);
	
	
	printf("Raiz1 = %.2f\n", raiz1);
	printf("Raiz2 = %.2f\n", raiz2);

	return 0;
}

File: Codigo_Fonte_pascal/lista1/lista1_12.pas
program segundo_grau;

var
   a, b, c, delta, raiz_delta, raiz1, raiz2 : real;

begin
     writeln('Digite os valores a, b, c :');
     readln(a);
     readln(b);
     readln(c);

     delta := sqr(b) - (4 * a * c);
     raiz_delta := sqrt(delta);
     raiz1 :=(-b + raiz_delta) / (2 * a);
     raiz2 :=(-b - raiz_delta) / (2 * a);

     writeln(raiz1:0:2);
     writeln(raiz2:0:2);
     readln;

end.
